### DISCUSSION TITLE
Add image annotation

### DISCUSSION THREAD

**Discussion by metasj** [knowledgefutures/pubpub#1378] [Ideas]
Let editors select a rectangle and add an annotation.

cf this Semantic Image Annotator extension:
https://www.mediawiki.org/wiki/Extension:Semantic_Image_Annotator

**Comment by gabestein** [comment]
Hi everyone — I'm going in and converting some of the longer-lived feature request/idea issues from Issues to Discussion threads. There's no exact heuristic about what belongs in the forum vs. what belongs as an issue, so when in doubt, please post wherever you see fit — we won't mind! But in general, we'd like to make Issues a space for things that we're committed to building/fixing, and Discussions a space for most everything else. So, if you're reporting a bug or requesting we make a straightforward change to an existing feature, go ahead and log an issue. Otherwise, Discussions is likely a better bet.

So, in this case, we're moving it to discussions and inviting anyone who's also interested in the idea to comment!

**Comment by metasj** [comment]
[Annotorious](https://recogito.github.io/annotorious/), building on openseadragon, is a nice simple place to start, and the resulting annotation text could be displayed in the side margin along with text notes.  

For larger images that would want zooming, see the discussion about an IIIF embed; those might only be viewable within the embed, for practical space reasons, but could have a number appear in the sidebar to indicate how much detail commentary is available.
